styleguide.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  components: 'components/**/*.js',
+  webpackConfig: {
+    module: {
+      rules: [
+        // Babel loader will use your project’s babel.config.js
+        {
+          test: /\.jsx?$/,
+          exclude: /node_modules/,
+          loader: 'babel-loader',
+        },
+        // Other loaders that are needed for your components
+        {
+          test: /\.css$/,
+          use: ['style-loader', 'css-loader'],
+        },
```

Comment:
We should figure out if we need this css part here as we are planning on using styled components. But I think we can keep it until then and figure it out on the go .

Reply:
Good point!